tests/test_maker_utils.py

```diff
@@ -144,9 +142,6 @@ def mutate_value(value):
         if isinstance(value, Time):
             return value + 1 * u.day
 
-        if isinstance(value, list):
```

Comment:
The docstring mentions that 'Pure lists need to be preserved'

https://github.com/spacetelescope/roman_datamodels/blob/e9d93fdef8c80f26036bd09a0404d9ba89a157fd/tests/test_maker_utils.py#L136-L148

Is this no longer necessary? I'm not familiar with this code so would you describe why it's no longer needed and if so, update the docstring?

Reply:
It was never needed.